codechain/keystore/CCkey.py

```diff
@@ -0,0 +1,44 @@
+from jsondb.db import Database
+
+from .KeystoreManager import keystoreManager
+from .KeyType import KeyType
+
+
+class CCkey:
+    def __init__(self, db_path):
+        self.platform = keystoreManager(KeyType.PLATFORM, db_path)
+        self.asset = keystoreManager(KeyType.ASSET, db_path)

```

Comment:
I think this is better design
```
db = Database(db_path)
db.data(dictionary={"meta": "", "platform": [], "asset": [], "hdwseed": []}) 
self.platform = keystoreManager(KeyType.PLATFORM, db)
self.asset = keystoreManager(KeyType.ASSET, db)
```
It clearly express that two keystoreManager shares same DB. Also jsondb provides same interface with the `dict`, it gives you more flexiblility to test `keystoreManager` without involving file operation.

Reply:
Yes I changed it.